src/platform/test/functional/page_objects/discover_page.ts

```diff
@@ -285,6 +285,12 @@ export class DiscoverPageObject extends FtrService {
     return await this.testSubjects.exists('unifiedHistogramChart');
   }
 
+  public async closeDatePickerPopover() {
```

Comment:
Nit: Seems like this method would be better suited to a common FTR service like `TimePickerPageObject` since it's not really Discover specific.